adapters/tappx/params_test.go

```diff
@@ -60,4 +65,13 @@ var invalidParams = []string{
 	`{"tappxkey": 1, "endpoint":"ZZ1INTERNALTEST149147915", "host":""}`,
 	`{"tappxkey":"pub-12345-android-9876", "endpoint": 1, "host":""}`,
 	`{"tappxkey": 1, "endpoint": 1, "host": 123}`,
+	`{"tappxkey": "1", "endpoint": 1}`,
+	`{"tappxkey": "1", "endpoint": 1}`,
```

Comment:
Please delete. This is a duplicate of the case above.

Reply:
I modified to add more cases in 8d09494 and now is not duplicated!
Thanks for all.